src/components/Edit.jsx

```diff
@@ -7,8 +7,8 @@ import {
 } from 'volto-subblocks';
 import { SidebarPortal } from '@plone/volto/components';
 
-import EditBlock from 'volto-form-block/components/EditBlock';
-import Sidebar from 'volto-form-block/components/Sidebar';
+import EditBlock from './EditBlock';
+import Sidebar from './Sidebar';
```

Comment:
@mamico imports should be left as they were, because otherwise there are problems with customizations and in some projects

Reply:
@giuliaghisini I don't know: without those changes my test design does not work ... please change back and test the component yourself.